recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_targets:
```

Comment:
Why is this needed?

Reply:
Hi @nicoddemus !
Thanks a lot for this review. This will publish the package with a `dev` label, making it not available for conda-forge as a stable release but available using the `dev` label!

Reply:
Ahh right, found the related docs:

https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=channel_targets#creating-a-pre-release-build

Thanks!